CMakeLists.txt

```diff
@@ -299,7 +299,10 @@ if(CMAKE_C_COMPILER_ID MATCHES "Clang")
 elseif(CMAKE_C_COMPILER_ID MATCHES "GNU")
     message(STATUS "libavif: Enabling warnings for GCC")
     add_compile_options(-Wall -Wextra)
-elseif(CMAKE_C_COMPILER_ID MATCHES "MSVC")
+elseif(MINGW)
+    add_compile_options(-municode)
+    add_compile_definitions(UNICODE _UNICODE)
```

Comment:
Delete this line.

Alternatively, add this line for all compilers on Windows. I suggested this before based on [GCC documentation of -municode](https://gcc.gnu.org/onlinedocs/gcc/x86-Windows-Options.html#index-municode):

> `-municode`
> 
> This option is available for MinGW-w64 targets. It causes the `UNICODE` preprocessor macro to be predefined, and chooses Unicode-capable runtime startup code.

The rationale is that if GCC defines the `UNICODE` preprocessor macro (when `-municode` is specified), then we should define that macro for other compilers on Windows, otherwise MINGW and other compilers will differ in this aspect.

But I now think we need to verify exactly what preprocessor macros `-municode` defines before we do this. So I suggest we just delete this line for now.